jupyter_server_nbmodel/handlers.py

```diff
@@ -227,7 +237,20 @@ async def _execute_snippet(
                 del ycell["outputs"][:]
                 ycell["execution_count"] = None
                 ycell["execution_state"] = "running"
-
+                if metadata.get("record_timing", False):
+                    time_info = ycell["metadata"].get("execution", {})
+                    time_info["shell.execute_reply.started"] = datetime.now(timezone.utc).isoformat()[:-6]
+                    ycell["metadata"]["execution"] = time_info
+            # Emit cell execution start event
+            event_logger.emit(
+                schema_id="https://events.jupyter.org/jupyter_server_nbmodel/cell_execution/v1",
+                data={
+                    "event_type": "execution_start",
+                    "cell_id": metadata["cell_id"],
+                    "document_id": metadata["document_id"],
+                    "timestamp": datetime.now(timezone.utc).isoformat()
```

Comment:
Wondering if we should not store the instant once and use it for setting the `shell.execute_reply.started` and `event.timestamp` to avoid slight time shift (and avoid requesting twice the current time).